subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/DefaultIvyArtifactRepository.java

```diff
@@ -311,6 +382,20 @@ void reset() {
             artifact = false;
         }
 
+        List<String> asList() {
```

Comment:
Should comment here on the impact of changing these strings.

Reply:
done